templates/run_template.sh

```diff
@@ -0,0 +1,24 @@
+PROJECT="ansible-service-broker"
+BROKER_IMAGE="ansibleplaybookbundle/ansible-service-broker:latest"
+OPENSHIFT_TARGET="https://kubernetes.default"
+OPENSHIFT_USER="admin"
+OPENSHIFT_PASS="admin"
+DOCKERHUB_USER="CHANGEME"
+DOCKERHUB_PASS="CHANGEME"
+DOCKERHUB_ORG="ansibleplaybookbundle"
```

Comment:
The benefit is now that the variables are specified as parameter you can use "oc process -p DOCKERHUB_ORG=foo | oc create -f -" to deploy, instead of a combination of sed scripts to modify the yaml.

Example of using a template is here: 
https://github.com/jwmatthews/ansible-service-broker/blob/template/templates/run_template.sh#L24